ex1/exceptions.py

```diff
@@ -37,6 +37,9 @@
 # Use the function search_n from the functions module inside a new function
 # also named search_n. The function should do the same as functions.search_n
 # but if the variable is not found in the list then raise a ValueError.
+def search_n(list, x):
+	return (list.index(x), x)
```

Comment:
The main point here was to work with functions that have the same name but are in different namespaces. So you should have reused the `search_n` function from the other module.

Avoid using Python builtins like `list` as variable names.